static/js/constants/app/explore_constants.ts

```diff
@@ -37,6 +37,21 @@ export const URL_HASH_PARAMS = {
   MAXIMUM_BLOCK: "mb",
   TEST_MODE: "test",
   I18N: "i18n",
+  CLIENT: "client",
+};
+export const CLIENT_TYPES = {
+  // User typed in a p=X&t=Y URL
+  ENTITY: "ui_entity",
```

Comment:
nit: should this be "ui_topic"? since this is a topic page?

Reply:
More or less all of the below is from topic page.  The CLIENT_TYPES (maybe I need a better name) indicate the specific context for it.  This is where user provided a url with entity, maybe I'll make it `ui_url_entity` if its clearer/